docs/reference/announcements-release-notes/870/870-announcements.md

```diff
@@ -1,32 +1,63 @@
 
-| Scheduled release date | Scheduled end of maintenance | Release notes                                        | Blog                                                                                            |
-| :--------------------- | :--------------------------- | :--------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
-| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
+| Scheduled release date | Scheduled end of maintenance | Release notes                                                                        | Blog                                                                                            |
+| :--------------------- | :--------------------------- | :----------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
+| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/announcements-release-notes/870/870-release-notes.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
 
 <!--- [Ad-hoc subprocesses](#)
 - [Document handling](#)
 - [RPA](#)
   - [Fetch RPA resource API](#)
   - [deployResourceAPI for RPA](#) -->
 
-- [Amazon Web Services (AWS) region support](#amazon-web-services-aws-region-support-saas)
-- [Web Modeler](#web-modeler)
-  - [Web Modeler Self-Managed deploy diagram change](#web-modeler-deploy-diagram-change)
+- [Versioning and environment changes](#versioning-and-environment-changes)
+- [Breaking changes](#breaking-changes)
+  - [Camunda Process Test](#camunda-process-test)
+  - [Deprecated: OpenAPI entities with `integer (int64)` key attributes](#deprecated-openapi-entities-with-integer-int64-key-attributes)
+- [New features](#new-features)
+  - [Amazon Web Services (AWS) region support SaaS](#amazon-web-services-aws-region-support-saas)
+  - [Deploy diagram change Self-Managed](#web-modeler-deploy-diagram-change)
   - [Milestones renamed to versions](#web-modeler-milestones-renamed-to-versions)
-- [Southeast Asia region for SaaS customers SaaS](#southeast-asia-region-for-saas-customers-saas)
-- [Breaking changes in Camunda Process Test](#breaking-changes-in-camunda-process-test)
-- [API updates](#api-updates-saasself-managed)
+  - [Southeast Asia region for SaaS customers SaaS](#southeast-asia-region-for-saas-customers-saas)
 
-## Amazon Web Services (AWS) region support <span class="badge badge--long" title="This feature affects SaaS">SaaS</span>
+## Versioning and environment changes
+
+:::note
+
+There are currently no known versioning changes or environment changes associated with the 8.7 release.
+
+:::
```

Comment:
I like this, but it may be difficult to maintain. If we keep this, we would need to add it to the docs release manager checklist and be sure to include instructions on Confluence for Engineers to know when to update this.